docs/docs-graphql/schema.graphqls

```diff
@@ -2791,6 +2801,13 @@ type Query {
   version: String!
 }
 
+"""Expected return type of an execution"""
+enum ReturnType {
+  Success
+  Failure
+  Any
```

Comment:
Yikes, yup. Got caught out by copy-pasting from above: https://github.com/dagger/dagger/pull/8669#discussion_r1820892962

Reply:
Should we prefix it with `Expect` to avoid having a super-generic type lying around? ie `EXPECT_ANY`, `EXPECT_SUCCESS` etc? Or do we plan on using these types for different purposes?

Reply:
They're scoped by the enum and #8669 is the fix that scopes the enum values to their enum types.

Reply:
We need to solve this in https://github.com/dagger/dagger/pull/8669

Reply:
A note on choosing `ReturnType` as a name instead of `Expect` in the name: I think there's potential applications in other places.

For example: we could have a `restartOn=ReturnType.Success` for a dagger service that would automatically restart a service if it stopped for some reason. Not saying we should do *that* feature, but `ReturnType` is more generic in other places than `ExpectType` or similar.

Reply:
To be fair, from an API POV the enum values are global it seems. So you won't be able to have another enum with the `ANY` value for example.

Reply:
Wait they're global? :scream: If so, that feels like a big issue.

Reply:
Phew, okay. I was operating under the assumption that you could have duplicate enum values at the graphql layer - you always know what type you need to unmarshal into, so it should be fine.